features/flows/elastic/src/main/java/org/opennms/netmgt/flows/elastic/ElasticFlowRepository.java

```diff
@@ -92,553 +92,575 @@
 
 public class ElasticFlowRepository implements FlowRepository {
 
-    public static final String OTHER_APPLICATION_NAME = "Other";
```

Comment:
Please revert the formatting changes here - this makes it difficult to see what actually changed.